src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -279,27 +279,36 @@ public bool HasNoResults
         }
 
         /// <summary>
-        /// Checks if the user already has a package by the given name.
+        /// Checks if the package can be installed.
         /// </summary>
         /// <param name="packageName"></param>
         /// <returns></returns>
-        internal bool UserAlreadyHasPackageInstalled(string packageName)
+        internal bool CanInstallPackage(string name)
         {
-            PackageManagerExtension packageManagerExtension = PackageManagerClientViewModel.PackageManagerExtension;
-            
-            // This check fires before the package is added to PackageLoader.LocalPackages collection.
-            // Therefore, we need to check both the names of downloaded packages and recent downloads.
-
-            List<string> loadedPackageNames = packageManagerExtension.PackageLoader.LocalPackages
-                .Where(x => x.LoadState.State == PackageLoadState.StateTypes.Loaded)
-                .Select(x => x.Name)
-                .ToList();
-
-            List<string> recentlyDownloadedPackageNames = Downloads
-                .Select(x => x.Name)
-                .ToList();
+            // Return true if there are no matching non built-in packages
```

Comment:
Why check for non-built-in only? Don't we need to check for no matching package among all packages?

Reply:
My thinking was that we should allow a user to install a package that already exists as a built-in. How else would the user be able to override an existing built-in package ?

Reply:
Right, but why not allow the user to also override a custom package?

Reply:
If by custom package you mean non-built in package, then the answer is in the original PR https://github.com/DynamoDS/Dynamo/pull/12033
Looks like the original requirement was to disable the Install button if a package is already installed

Reply:
Yes by custom package I mean non-built-in package. We are using custom package as the term to describe these types of packages, which is what you'll see in the toggles.